### DISCUSSION TITLE
Why does Front Matter want to access my GitHub account in VSCode?

### DISCUSSION THREAD

**Discussion by stv8** [estruyf/vscode-front-matter#510] [Q&A] [answered]
I just noticed this notification from VSCode about Front Matter wanting to access my GitHub account.

I tried searching but couldn't find anything mentioning what it needs to access my account for.

Is this simply due to me having "settings sync" enabled which ties my VSCode settings to GitHub?

<img width="310" alt="image" src="https://user-images.githubusercontent.com/5158539/219969461-8f800639-470d-4e96-ae40-dde527d5095b.png">

Thanks!

**Comment by estruyf** [comment, marked as answer]
Hi @stv8, thank you for asking. We only use this to hide the sponsor links in the CMS, as that is the only thing I can give back. It checks your username against the API, which knows checks the sponsors. If you are a known sponsor, it will hide these sponsor elements.

**Comment by Welsh44** [comment]
Hi @estruyf 
Thanks for the answer.
By the way is there a way to disable this functionnality in order to hide the notification ?
Thanks
